integration-test/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "integration-test",
+  "version": "1.0.0",
```

Comment:
is there anything we can add to this package.json to prevent it from being accidentally published? Or should we just set the name to be something obviously scoped to the Nat package?

Reply:
Yep! We can add "private": true. https://docs.npmjs.com/files/package.json#private